model/src/main/java/org/cbioportal/model/GenesetCoExpression.java

```diff
@@ -0,0 +1,59 @@
+package org.cbioportal.model;
```

Comment:
Seems like this file is not used anywhere?

Reply:
@ersinciftci Looks like this package needs to be declared.